_getting-started/security.md

```diff
@@ -0,0 +1,192 @@
+
+opensearch_security.multitenancy.enabled: true
+opensearch_security.multitenancy.tenants.preferred: [Private, Global]
+opensearch_security.readonly_mode.roles: [kibana_read_only]
+# Use this setting if you are running opensearch-dashboards without https
+opensearch_security.cookie.secure: false
+```
+{% include copy.html %}
+
+You can start the binary or service, depending on which method was used to install OpenSearch and OpenSearch Dashboards.
+
+When using binary, you need to supply `--no-base-path` to `yarn start` command to set a URL without a base-path. If this is not set, a random 3-letter base-path will be added.
+{: .note}
+
+After OpenSearch Dashboards is started, you should see following two lines in the logs:
+
+```
+[info][listening] Server running at http://localhost:5601
+[info][server][OpenSearchDashboards][http] http server running at http://localhost:5601
+```
+{% include copy.html %}
+
+You can now access the OpenSearch Dashboards using http://localhost:5601 in your browser. Use the username `admin` and the password that was configured in `OPENSEARCH_INITIAL_ADMIN_PASSWORD` environment variable.
+
+# Adding users
+
+There are three ways to add users, roles, and other security related configurations:
+
+  - Updating appropriate configuration files (`internal_users.yml` file for adding/updating/removing users) 
+  - Using the API
+  - Using the OpenSearch Dashboards UI
+
+Security configuration files are located in `config/opensearch-security` directory.
+{: .note}
+
+You can add an OpenSearch Dashboards user by updating the `internal_users.yml` file with the following settings:
+
+```
+test-user:
+  hash: "$2y$12$CkxFoTAJKsZaWv/m8VoZ6ePG3DBeBTAvoo4xA2P21VCS9w2RYumsG"
+  backend_roles:
+  - "test-backend-role"
+  - "kibanauser"
+  description: "test user user"
+```
+{% include copy.html %}
+
+The `hash` string is generated using `hash.sh` script located in `plugins/opensearch-security/tools/` directory. In this case the hash of the string `secretpassword` was used.
+
+Note the use of built-in backend role `kibanauser` which is going to give user permissions needed to navigate OpenSearch Dashboards.
+
+## Creating roles
+
+Roles inside of `roles.yml` use the following structure:
+
+```
+<rolename>:
+  cluster_permissions:
+    - <cluster permission>
+  index_permissions:
+    - index_patterns:
+      - <index pattern>
+      allowed_actions:
+        - <index permissions>
+```
+{% include copy.html %}
+
+Using this structure you can configure a new role to give access to specific indexes, such as the following example configuration:
+
+```
+human_resources:
+  index_permissions:
+    - index_patterns:
+      - "humanresources"
+      allowed_actions:
+        - "READ"
+```
+{% include copy.html %}
+
+Note that the cluster permissions are not listed in this example, as these are provided by built-in role `kibana_user` which is already mapped using the `kibanauser` backend role.
```

Comment:
```suggestion
Note that the cluster permissions are not listed in this example because these are provided by the built-in role `kibana_user`, which is already mapped using the `kibanauser` backend role.
```